changelogs/fragments/5281-locale_gen.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - "locale_gen - fix ubuntumode (https://github.com/ansible-collections/community.general/issues/5281, https://github.com/ansible-collections/community.general/pull/5281)."
```

Comment:
```suggestion
  - "locale_gen - fix support for Ubuntu (https://github.com/ansible-collections/community.general/issues/5281, https://github.com/ansible-collections/community.general/pull/5281)."
```

Reply:
OK. I've done it.
`4b581b5b349de7df1b4d3231a99d33f9caa60a47`
:)